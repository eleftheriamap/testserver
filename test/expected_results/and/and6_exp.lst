
test/exp/test/gen/and/and6_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	513a6a31 	sub	w17, w17, #0xe9a
   4:	d1334ad6 	sub	x22, x22, #0xcd2
   8:	0a567232 	and	w18, w17, w22, lsr #28
   c:	8a000000 	and	x0, x0, x0
