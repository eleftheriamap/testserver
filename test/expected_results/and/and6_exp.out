Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 00000000fffff166
X18    = 0000000000000006
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = fffffffffffff32e
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 000000000000000c
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0x513a6a31
0x00000004 : 0xd1334ad6
0x00000008 : 0x0a567232
0x0000000c : 0x8a000000
